BUILD.md

```diff
@@ -83,7 +83,13 @@ list some basic and more used commands as example.
     ```
 
     What this does is to install to a temporary location and calling
```

Comment:
While at it, I think it's worth rewording this telling that the `--upload` switch does this. Otherwise, this might give the idea that the *configure* command does this.